doc(release): add release notes for collect 23.10.5 (#3159)
centreon/centreon-documentation: versioned_docs/version-23.10/releases/centreon-os.mdx

## Centreon Collect

### 23.10.5

Release date: `March 19, 2024`

#### Centreon Broker

<details open>
  <summary>Bug Fixes</summary>

> - [Broker] During BA initializations, kpi could be set with incoherent status. This was breaking BAs.

Reviewer: ```suggestion
- [Broker] BA initialization could result in KPIs having incoherent statuses. This was breaking the statuses of the BAs.
```
